api/graphql/reservation_units/reservation_unit_serializers.py

```diff
@@ -91,48 +91,59 @@ class Meta(PurposeCreateSerializer.Meta):
 
 
 class ReservationUnitCreateSerializer(ReservationUnitSerializer, PrimaryKeySerializer):
-    terms_of_use_fi = serializers.CharField(required=False)
-    terms_of_use_sv = serializers.CharField(required=False)
-    terms_of_use_en = serializers.CharField(required=False)
-    name_fi = serializers.CharField(required=False, allow_blank=True)
-    name_sv = serializers.CharField(required=False, allow_blank=True)
-    name_en = serializers.CharField(required=False, allow_blank=True)
-    max_reservation_duration = DurationField(required=False)
-    min_reservation_duration = DurationField(required=False)
-    buffer_time_before = DurationField(required=False)
-    buffer_time_after = DurationField(required=False)
-    max_persons = serializers.IntegerField(required=False)
+    terms_of_use_fi = serializers.CharField(required=False, allow_null=True)
+    terms_of_use_sv = serializers.CharField(required=False, allow_null=True)
+    terms_of_use_en = serializers.CharField(required=False, allow_null=True)
+    name_fi = serializers.CharField(required=False, allow_null=True, allow_blank=True)
+    name_sv = serializers.CharField(required=False, allow_null=True, allow_blank=True)
+    name_en = serializers.CharField(required=False, allow_null=True, allow_blank=True)
+    max_reservation_duration = DurationField(required=False, allow_null=True)
+    min_reservation_duration = DurationField(required=False, allow_null=True)
+    buffer_time_before = DurationField(required=False, allow_null=True)
+    buffer_time_after = DurationField(required=False, allow_null=True)
+    max_persons = serializers.IntegerField(required=False, allow_null=True)
     space_pks = serializers.ListField(
         child=IntegerPrimaryKeyField(queryset=Space.objects.all()),
         source="spaces",
         required=False,
+        allow_null=False,
```

Comment:
Should we allow nulls to here? and also to `resource_pks`, `purpose_pks`, `equipment_pks`,  `service_pks`

Reply:
Good question. I double checked this -- we don't want to allow `null` here.

Since the value is returned as an empty list (not `null`) in queries, we also don't need to allow setting it to `null`.

However, the explicit `allow_null=False,` was left here by mistake. 😳 I removed it from this and the other fields.